tests/datasets/api_tests.py

```diff
@@ -870,6 +870,9 @@ def test_export_dataset(self):
             if export_table["table_name"] == "birth_names":
                 expected_response = export_table
                 break
+        else:
```

Comment:
On other places we are using `if backend() == "hive":` then skip test for example